src/EventsTypes.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import * as Types from "./Types";
+
+export interface TabsterMoveFocusEventDetail {
+    by: "mover" | "groupper" | "modalizer" | "root" | "deloser";
+    owner: HTMLElement; // Mover, Groupper, Modalizer or Root, the initiator.
+    next: HTMLElement | null; // Next element to focus or null if Tabster wants to go outside of Root (i.e. to the address bar of the browser).
+    relatedEvent?: KeyboardEvent; // The original keyboard event that triggered the move.
+}
+
+export type MoverMoveFocusEventDetail = { key: Types.MoverKey };
+
+export interface MoverMemorizedElementEventDetail {
+    memorizedElement: HTMLElement | undefined;
+}
+
+export type GroupperMoveFocusEventDetail = {
```

Comment:
why is there a mix of `type` and `interface`?

Reply:
This is likely a leftover of event typings evolution. Adjusted to interfaces.